roscala/src/main/scala/coop/rchain/rosette/Location.scala

```diff
@@ -52,9 +52,9 @@ object Location {
 
               case GlobalVariable(offset) => pureCtxt(globalEnv.slot.lift(offset))
 
-              case BitField(indirect, level, offset, spanSize, sign) => pureCtxt(None)
+              case BitField(_, _, _, _, _) => pureCtxt(None)
```

Comment:
More of a personal preference, but I think `case _: BitField` would be better. If you're not going to use any of the extracted values, why do the extraction at all?

Reply:
Done